### PR TITLE
#874 pytest added as test framework

### PR DESCRIPTION
**Pull request by mouseratti**

### CODE REVIEW

**Comment by mouseratti** [comment]
@Stibbons could you review it, please?

File: setup.py [hunk 1]
```diff
@@ -7,6 +7,7 @@
 from setuptools import setup
 
 setup(
-    setup_requires=['pbr'],
+    setup_requires=['pbr', 'pytest-runner', 'wheel'],
```
Comment (Stibbons) [inline setup.py]: why don't you use `requirements-dev.txt`?

File: setup.py [hunk 2]
```diff
@@ -7,6 +7,7 @@
...
+    tests_require=['pytest', ],
```
Comment (Stibbons) [inline setup.py]: isn't there a way to insert it into setup.cfg and keep setup.py as minimal as possible?

Also, useless comma. `tests_require=['pytest']` works.

File: setup.cfg [hunk 3]
```diff
@@ -24,7 +24,6 @@ keywords =
 [files]
 packages =
     guake
-
```
Comment (Stibbons) [inline setup.cfg]: please keep this line

**Comment by mouseratti** [comment]
current PR is closed, see #880

**Pull request closed without merging**